test/integration_test_helper.rb

```diff
@@ -203,9 +203,10 @@ def save_edition_and_assert_success
     end
   end
 
-  def save_edition_and_assert_error
+  def save_edition_and_assert_error(error: nil)
```

Comment:
```suggestion
  def save_edition_and_assert_error(error = nil)
```

I think this will read better this way, compare: `save_edition_and_assert_error("can't be blank")` with `save_edition_and_assert_error(error: "can't be blank")`